django_mailbox/models.py

```diff
@@ -351,10 +351,12 @@ def _process_message(self, message):
         if settings['store_original_message']:
             self._process_save_original_message(message, msg)
         msg.mailbox = self
+        # Fix to accept subject emojis in utf-8
         if 'subject' in message:
             msg.subject = (
-                utils.convert_header_to_unicode(message['subject'])[0:255]
+                utils.convert_header_to_unicode(unicode(message['subject']).decode('utf-8'))[0:255]
```

Comment:
This is a rather surprising change; could you elaborate on how this helps, exactly?

Reply:
I am working in a app with python 2.7, dango 2.11 and production database in 'utf8' charset. And i need to use django-mailbox to receive emails. If an email have a 'emoji' in subject, Django return a OperationalError. I should not change the character set to 'utf8mb4' in production. This fix (I don't know another way to do it, in utils.convert_header_to_unicode perhaps?) allow receive emails with emojis in django 1.11, python 2.7 and utf8 charset and collation